.ci/run_tests_minikube_linux.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# NOTE: This script assumes that minikube is installed and running, and using the docker driver on Linux
+# Due to networking issues with the docker driver and ingress on macOS/Windows, this script must be run on Linux
+
+# Share docker env with Minikube
+eval $(minikube docker-env)
+
+# error on unset variables
+set -u
+# print each command before executing it
+set -x
+
+# Build the test devfile registry image
+./build_registry.sh
```

Comment:
Should we catch the error and exit here if the image build process is failed instead of continuing the deploy process?

Reply:
Yes, that should be caught.

Reply:
Updated.